pkg/openai/openai.go

```diff
@@ -0,0 +1,54 @@
+package openai
+
+import (
+	"context"
+)
+
+type Client struct {
+	apiKey string
+	model  string
+	debug  bool
+}
+
+const promptBoilerplate = `
+Here is an example OPA policy. Can you write a brand new OPA policy example using the 'fairwinds' package?
+
+<>
+package fairwinds
+
+labelblock[actionItem] {
+    # List the keys of Kubernetes labels that will be blocked.
+    blockedLabels := {"foo"}
+    provided := {label | input.metadata.labels[label]}
+    missing := blockedLabels - provided
+    found := blockedLabels - missing
+    count(found) > 0
+    actionItem := {
+        "title": "Bad label is present",
+        "description": sprintf("Label %v is present", [found]),
+        "severity": 0.1,
+        "remediation": "Remove the label",
+        "category": "Reliability"
+    }
+}
+<>
+
+The new policy you write should: '`
```

Comment:
Is there an extra singlequote here?
```suggestion
The new policy you write should: `
```